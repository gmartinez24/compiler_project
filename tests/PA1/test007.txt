// Grant Martinez
// Testing multiple types of errors mixed in with valid tokens

main

int dollarInFront             = $1

float forgotToFinishThisFloat = 345.

float floatCombinedWithWord   = 24.five

float floatWithInvalid        = 98.~

int invalid_name@23 = 12
