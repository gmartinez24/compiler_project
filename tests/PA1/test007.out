MAIN
INT
IDENT	dollarInFront
ASSIGN
ERROR
INT_VAL	1
FLOAT
IDENT	forgotToFinishThisFloat
ASSIGN
ERROR
FLOAT
IDENT	floatCombinedWithWord
ASSIGN
ERROR
IDENT	five
FLOAT
IDENT	floatWithInvalid
ASSIGN
ERROR
INT
IDENT	invalid_name
ERROR
INT_VAL	23
ASSIGN
INT_VAL	12
EOF